src/test/jtx/Env.h

```diff
@@ -79,6 +79,18 @@ features (uint256 const& key, Args const&... args)
     return {{key, args...}};
 }
 
+// two tag types used to invoke specializations
+// of construct_arg that will modify the configuration
+struct admin_t {
+    admin_t(bool is_admin) : is_admin(is_admin) {};
+    bool is_admin;
```

Comment:
The rules are relaxed for jtx code since its not part of the implementation of rippled.